src/calculateRentalCost.js

```diff
@@ -29,6 +29,17 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let total = 0;
+
+  if (days >= 7) {
+    total = days * 40 - 50;
+  } else if (days >= 3) {
+    total = days * 40 - 20;
+  } else {
+    total = days * 40;
```

Comment:
please add variables for those numbers - [checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md) point 2